src/modules/tools/laser/Laser.cpp

```diff
@@ -59,8 +61,14 @@ void Laser::on_module_loaded() {
     this->laser_pin->period_us(THEKERNEL->config->value(laser_module_pwm_period_checksum)->by_default(20)->as_number());
     this->laser_pin->write(this->laser_inverting ? 1 : 0);
 
-    this->laser_max_power =    THEKERNEL->config->value(laser_module_max_power_checksum   )->by_default(0.8f)->as_number() ;
-    this->laser_tickle_power = THEKERNEL->config->value(laser_module_tickle_power_checksum)->by_default(0   )->as_number() ;
+    this->laser_max_power = THEKERNEL->config->value(laser_module_max_power_checksum   )->by_default(1.0f)->as_number() ;
```

Comment:
This changes the defualt behavior and therefore breaks backwards compatibility with legacy config files.